packages/@n8n/benchmark/scripts/n8n-setups/scaling-single-main/docker-compose.yml

```diff
@@ -117,10 +139,6 @@ services:
       - DB_TYPE=postgresdb
       - DB_POSTGRESDB_HOST=postgres
       - DB_POSTGRESDB_PASSWORD=password
-      # Task Runner config
-      - N8N_RUNNERS_ENABLED=true
-      - N8N_RUNNERS_MODE=internal
-      - N8N_NATIVE_PYTHON_RUNNER=true
```

Comment:
To simplify, I decided to remove all runner config from mains in `scaling-single-main` and `scaling-multi-main` as only workers need access to runners in scaling setups.